package.json

```diff
@@ -20,13 +20,14 @@
   "license": "ISC",
   "devDependencies": {
     "concurrently": "^4.1.0",
-    "nodemon": "^1.18.7",
-    "mysql2": "^1.6.5"
+    "mysql2": "^1.7.0",
+    "nodemon": "^1.18.7"
   },
   "dependencies": {
     "babel-eslint": "^10.0.1",
     "express": "^4.17.1",
     "if-env": "^1.0.4",
+    "mysql": "^2.18.1",
```

Comment:
we don't need this in production as we will be using a postgres database.

Reply:
Thank you for the review! I will remove this package

Reply:
I just removed the package!